slidey_js.js

```diff
@@ -0,0 +1,101 @@
+$( document ).ready(function() {
+    console.log( "ready!" );
+});
+  var background = $("#background")
+  enMarche = 0
+
+  //
+  // var getAPI = function() {
+  //   var giphyAPI = "http://api.giphy.com/v1/gifs/search?q=ryan+gosling&api_key=dc6zaTOxFJmzC&limit=5";
+  //   $.getJSON( giphyAPI, {
+  //     // tags: "mount rainier",
+  //     // tagmode: "any",
+  //     format: "json"
+  //   })
+  // }
+  //
+  //
+  // var xhr = $.getJSON("http://api.giphy.com/v1/gifs/search?q=ryan+gosling&api_key=dc6zaTOxFJmzC&limit=5");
+  // console.log(xhr);
+  // xhr.done(function(data) { console.log("success got data", data); });
+
+  var imageArray = ["http://unidevphp.com/php-projects/penymobile/images/auction_images/7_564901_images.jpg", "http://i.dailymail.co.uk/i/pix/2015/01/06/2473100D00000578-2898639-Photographer_Andrey_Gudkov_snapped_the_images_on_the_plains_of_t-a-20_1420546215677.jpg", "http://www.zastavki.com/pictures/originals/2013/Photoshop_Image_of_the_horse_053857_.jpg", "http://www.hdwallpapersimages.com/wp-content/uploads/2014/02/Magic-Of-Blue-Universe-Images.jpg"]
+
+  var counter = 0;
+  var changeBackground = function(){
+    if (enMarche = 1){
+      $("#background").css("background-image", "url('" + imageArray[counter] + "')")
+      console.log(imageArray[counter])
+      counter = counter + 1
+      if (counter >= imageArray.length){
+        counter = 0;
+      }
+    }
+  }
+
+  var startImageSwap = function(){
+    enMarche = 1
+    changeBackground();
+    console.log("bob")
+    interval = setInterval(changeBackground, 1000);
+  };
+
+  startImageSwap();
+
```

Comment:
I imagine you tried to do the logic for the starting and stopping of the slideshow in this function. I think that the starting and stopping of the slide show will have more to do with the click event itself.